### DISCUSSION TITLE
Scrum Prep: Milestone 3 Week 3

### DISCUSSION THREAD

**Discussion by YanehRuiz** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#407] [Weekly Retrospective]
Weekly Development Update: Third week of Milestone 3

Hello team!

Please provide a brief comment detailing what you accomplished this week, including any challenges faced or insights gained. Make sure to submit your updates by the deadline, which is before: Friday, November 15th, 2024 @ 11 am 

Thank you for your collaboration!

As for my collaboration:

Approved, requested edits and graded points on the issues and discussed the teams weekly progress.

**Comment by JorgeC08** [comment]
Completed my second LTT and I'm already drafting my two remaining ones.

**Comment by ghost** [comment]
Completed sprint issue, now users are able to read guidelines before submitting a report. Will work on other tasks as I can.

**Comment by Jjulianirizarry24** [comment]
Fixed a merge conflict, communicated with team.

**Comment by CaribbeanCool** [comment]
Finished a documentation issue

**Comment by noel-vargas** [comment]
Started implementing the automated scans with Zap

**Comment by javier-rivera8** [comment]
Started working with the lecture topic tasks on refactoring the edit profile code.

**Comment by JorgeRivera94** [comment]
Updated report labels and app colors to match designs. Have not been able to do more because of other parcial exams this week.

**Comment by johnsantiago9** [comment]
Worked on first sprint task and started developing new tasks

**Comment by misaelmercado1** [comment]
completion of my first ltt and researching the last one , and working on issues in the documentation

**Comment by JuanJRivera** [comment]
.This week I had hellweek and could not work on the proyect.

**Comment by Alejandro-DeJesus** [comment]
This week I worked on issue #396, but after that slowed down to focus on other classes.

**Comment by RaquelQuinones** [comment]
Working on the lecture topic task.

**Comment by joel-alvarado** [comment]
Added admin verification to admin-only routes in the database

**Comment by jonVargas1** [comment]
Couldn't work on the project this week due to other work

**Comment by NathBat09** [comment]
Research for Lecture Topic Tasks

**Comment by williammartinez10** [comment]
Brainstormed Lecture Topic Tasks to focus on over the upcoming weeks

**Comment by lex939** [comment]
Have been delegating and approving Documentation tasks posted (mainly feedback ones)

More issues to be posted soon

**Comment by Rodal1** [comment]
Brainstorming new issues, haven't had much time this week.

**Comment by Chosen-Juan1** [comment]
Not much progress, been rather busy this week.
